tests/Unit/PointwiseFunctions/AnalyticData/GrMhd/Test_BlastWave.cpp

```diff
@@ -149,7 +171,7 @@ void test_density_on_and_near_boundaries() {
 
   BlastWaveProxy cylindrical_blast_wave(
       inner_radius, outer_radius, inner_density, outer_density, inner_pressure,
-      outer_pressure, magnetic_field, adiabatic_index);
+      outer_pressure, magnetic_field, adiabatic_index, geometry);
```

Comment:
if `geometry` is specifiable, the variable should be renamed `cylindrical_blast_wave` -> `blast_wave`

Reply:
ping on this one